components/style/values/computed/mod.rs

```diff
@@ -587,3 +587,49 @@ impl ClipRectOrAuto {
 
 /// <color> | auto
 pub type ColorOrAuto = Either<Color, Auto>;
+
+/// <length> | <number> | <percentage> for SVG stroke-*
+#[allow(missing_docs)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+#[derive(Clone, Debug, PartialEq, ToCss)]
+pub struct SVGStrokeLength(pub NumberOrPercentage);
+
+impl ToComputedValue for specified::SVGStrokeLength {
+    type ComputedValue = SVGStrokeLength;
+
+    #[inline]
+    fn to_computed_value(&self, context: &Context) -> SVGStrokeLength {
+        match *self {
+            specified::SVGStrokeLength::LengthOrPercentage(ref lop) =>
+            {
+                match lop.to_computed_value(context) {
+                    LengthOrPercentage::Length(length) => {
+                        SVGStrokeLength(NumberOrPercentage::Number(length.to_f32_px()))
+                    },
+                    LengthOrPercentage::Percentage(percentage) => {
+                        SVGStrokeLength(NumberOrPercentage::Percentage(percentage))
+                    },
+                    LengthOrPercentage::Calc(_) => {
+                        // TODO: We need to support calc()
+                        panic!("Unexpected value.");
```

Comment:
This is kind of unfortunate... If this is something we can't fix now, is there a bug or issue for it?